src/lib/pure/ketrew_edsl.mli

```diff
@@ -251,3 +252,56 @@ val pbs :
   [> `Long_running of string * string ]
 (** Create a “PSB” build process. *)
 
+
+val yarn_application :
+  ?host:Ketrew_host.t ->
+  ?daemonize_using:[ `Nohup_setsid | `Python_daemon ] ->
+  ?daemon_start_timeout:float ->
+  Ketrew_program.t -> [> `Long_running of string * string ]
+(** Create a build process that requests resources from Yarn, the
+    command must be an application in the Yarn sense (i.e.
+    a program that is going to contact Yarn by itself to request
+    containers):
```

Comment:
Why is this a "build" process? That term is usually associated with something that compiles code.
How about:
Create a process that requests resources from Yarn. The command must be an application that contacts Yarn by itself to request containers: